### PR TITLE
Update build dependencies

### PR DESCRIPTION
**Pull request by friederbluemle**
- Gradle wrapper `3.4.1`
- Android Gradle plugin `2.3.1`
- build tools `25.0.2`
- support libs `25.3.1`

Note this also simplifies the way the support lib version is specified.

<sub><sup>Please NO GitHub rebase/squash merges. If you'd like me to combine some or all of the commits, please let me know, and I will update my branch. Thank you.</sup></sub>

### CODE REVIEW

**Comment by sockeqwe** [comment]
Thanks! Please update .travis.yml file to match build tools
https://github.com/sockeqwe/mosby/blob/master/.travis.yml#L6

Btw. Why no squash?

**Comment by friederbluemle** [comment]
@sockeqwe Sorry, forgot about .travis.yml - I just amended my commit.

Regarding squash/rebase on GitHub: Using this new feature will *rewrite* the commits on the GitHub server, i.e. the commit hash will change, and what ends up in the repo is not what I have in my local branch. This will make it necessary to *force delete* my local branch, even after merging all changes into master (Git will not see the branch as fully merged). Granted, using `-D` instead of `-d` when deleting the branch is just a minor inconvenience. Slightly more unsettling in that regard is the fact that without manually checking every line, I cannot be 100% sure that all of my force deleted changesets made it into the repo.
Bottom line is, if you want me to squash some/all of the commits, I'll gladly do so, but I'd rather perform the Git operation myself, and force push the updated branch. :)

**Comment by sockeqwe** [comment]
Thanks for the explanation and for your contribution!

**Comment by friederbluemle** [comment]
You're welcome.. btw, you used a squash merge (I think there is a little arrow next to the merge button to change it to regular merge). This is what I see in my local repo now (on the latest `master` branch):
```
$ git branch -d update-project 
error: The branch 'update-project' is not fully merged.
If you are sure you want to delete it, run 'git branch -D update-project'.
```

Next time, please let me know if you want me to squash the commits. Thanks :)

**Comment by sockeqwe** [comment]
Damn ... I didn't do this on purpose ... Sorry

Frieder Bluemle <notifications@github.com> schrieb am Mo., 10. Apr. 2017,
17:32:

**Comment by friederbluemle** [comment]
No worries :) Thanks for this awesome project :+1:
